cli/events-plugin.md

```diff
@@ -0,0 +1,328 @@
+
+> **Note:** Creating a project will also create a Production workspace that will be used later in this workflow when setting up credentials.
+
+Once the project has been created, add the *I/O Management API* service to the project. This will add the required scope to the authentication token required by the Events SDK. For a step-by-step guide to adding an API to a project, follow this tutorial for [adding an API to a project using Service Account (JWT) authentication](https://www.adobe.com/go/devs_projects_jwt) (the type of authentication used by the I/O Management API). 
+
+> **Note:** If you plan to use the [Events Templates](template-generator.md), you must also [add the Runtime Service](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/services-enable-runtime.md) to your project. This will ensure that the auth and namespace for the Runtime actions is added to your configuration.
+
+![](../img/cli-console-setup.png)
+
+### Setting up credentials using `aio console`
+
+After completing the setup within Console, you must then set up the credentials locally in order to run the various CLI commands. This set up will populate the `~/.config/aio` file with the access token, organization ID, workspace, and project details, as well as other credentials, required by the CLI. 
+
+On entering any command, you will be taken to the Adobe I/O CLI Login page. After signing in using your ID, you will be redirected to a page stating that you have successfully logged in to the Adobe I/O CLI. 
+
+![](../img/cli-adobe-io-cli-login.png)
+
+You can now proceed to select the organization, project, and workspace using the following steps:
+
+1. Select the same organization in which you created the project in Adobe Developer Console. You can list the organizations and select one using ***aio console org list***. 
+
+    ![](../img/cli-org-list.png)
+
+    To select an organization, use ***aio console org select <ims_org_id>***.
+
+    ![](../img/cli-org-select.png)
+
+
+2. Select the project you created in Console. You can list the projects using ***aio console project list***.
+
+    ![](../img/cli-project-list.png)
+
+    To select a project, use ***aio console project select <project_id>***.
+
+    ![](../img/cli-project-select.png)
+
+3. Select the workspace from the project you created in Console. By default a Production workspace is created when you create a new project. You can list the workspaces using ***aio console workspace list***.
+
+    ![](../img/cli-workspace-list.png)
+
+    To select a workspace, use ***aio console workspace select <workspace_id>***.
+
+    ![](../img/cli-workspace-select.png)
+
+## Installing and using the plugin
```

Comment:
I was thinking about this. They can just install aio cli and events plugin support is there in aio cli. This installation is only if they want to set up the plugin alone. https://github.com/adobe/aio-cli
So we can probably we can mention that they need to install the aio cli and then get started on the commands.

Reply:
Probably we can have something like this https://github.com/AdobeDocs/adobeio-events/pull/49/files?file-filters%5B%5D=.md#diff-3e589a59218311897461fda58100b0d3R18 here as well

Reply:
@sangeetha5491, I wasn't totally sure on the wording, but I tried to update this based on your feedback, please let me know if additional edits are needed. Thank you!